PIL/Image.py

```diff
@@ -1288,7 +1288,13 @@ def paste(self, im, box=None, mask=None):
         images (in the latter case, the alpha band is used as mask).
         Where the mask is 255, the given image is copied as is.  Where
         the mask is 0, the current value is preserved.  Intermediate
-        values can be used for transparency effects.
+        values will mix the two images together.  If this image has an
+        alpha channel, intermediate values will also set the alpha
+        channel to themselves.
+
+        See :py:meth:`~PIL.Image.Image.alpha_composite` if you want to
+        combine images with a partial mask without setting the alpha
```

Comment:
> without setting the alpha channel

That's not whole truth. If the background image has transparency, the result transparency also will be affected.